website/docs/r/service_connector_app_service.html.markdown

```diff
@@ -0,0 +1,133 @@
+---
+subcategory: "App Service (Web Apps)"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_connection"
+description: |-
+  Manages a service connector for app service.
```

Comment:
this doesn't match the PR title?

Reply:
Actually this PR contains 2 new resources. PR title modified.